src/Board.java

```diff
@@ -34,9 +37,28 @@ public void addShip(int x, int y, ShipType type) throws Exception {
 	     * statement where if the coordinate x and y are 7 then add ship.*/
```

Comment:
I tried to us if statements to pass the addShip test. I had a difficult time figuringout the coordinates.